alibi/datasets/default.py

```diff
@@ -49,7 +48,7 @@ def fetch_imagenet_10(url_id: int = 0) -> Dict:
 
     Parameters
     ----------
-    url_id
+    url_id: Index specifying which URL to use for downloading.
```

Comment:
This is not the `numpy` docstring format, description should be on a new line with an indent and no colon. If in doubt, recreate a new docstring which will auto-format (assuming Pycharm is set up with the `numpy` style docstrings) with all the parameters included.